app/assets/stylesheets/components/success_stories.scss

```diff
@@ -20,67 +23,100 @@
     margin-top: 100px;
     display: flex;
     flex-wrap: wrap;
-
+    @media(max-width: 480px){
+      margin-top: 20px;
+    }
     .success-story {
       flex-basis: 49.9%;
       display: flex;
-      .photo {
-        flex-basis: 100px;
-        text-align: center;
-        padding-top: 16px;
-
-          img {
-            width: 50px;
-            height: 50px;
-            border-radius: 50%
-          }
-        }
-        @media (max-width: 480px){
+      @media (max-width: 480px){
           &.story-2 {
             display: none;
           }
           &.story-3 {
             display: none;
           }
         }
-
-        .success-content {
-          flex: 1
+      padding-bottom: 36px;
+      .name-and-photo {
+        flex: 2;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        @media(max-width: 480px){
+          height: 90px;
         }
+      }
+      .photo1 {
+        flex-basis: 100px;
+        text-align: center;
+        padding-top: 16px;
+        display: flex;
+        justify-content: center;
+        img {
+          width: 50px;
+          height: 50px;
+          border-radius: 50%
+        }
+        @media(max-width: 480px){
+          display: none;
+        }
+      }
+      .photo2 {
+        display: none;
+        flex-basis: 60px;
+        text-align: center;
+        img {
+          width: 50px;
+          height: 50px;
+          border-radius: 50%
+        }
+        @media(max-width: 480px){
+          display: block;
+        }
+      }
+      .success-content {
+        flex: 1;
+      }
+      @media(max-width: 480px) {
+        padding: 0 26px 0 26px;
+      }
     }
-
- }
+  }
 
   .success-stories-read-more {
     width: 100%;
     text-align: center;
-
+    padding-top: 30px;
     a {
       color: rgba(74, 74, 74, 0.7);
```

Comment:
You didn't modify this line but can you please replace the color with the variable `$grey`?